org.gramps_project.Gramps.desktop

```diff
@@ -0,0 +1,146 @@
+[Desktop Entry]
```

Comment:
This ideally should be maintained in Gramps Project repository as it can be used by other distributions/stores.

Reply:
I just noticed it already is; feel free to handle it later on.

Reply:
I had to modify the desktop file to work with the flatpak, and as I understand it needs to be in a specific place for flathub.  So that is why there is a separate desktop file being moved in the flatpak manifest.  I am confused about what you want me to do.